docs/node-mixin/config.libsonnet

```diff
@@ -50,8 +50,8 @@
     // Available disk space (%) thresholds on which to trigger the
     // 'NodeFilesystemAlmostOutOfSpace' alerts.
     // 2 thresholds are available : high and low
```

Comment:
Great. Now you only need to adjust the comment here (it still says "high and low" – I'd say the naming is so expressive now that you can probably delet this line 52 entirely).

Reply:
done. Sorry.